src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -230,7 +256,82 @@ def excepthook(args):
 
         # If thunderscope is disabled, just run the test
         else:
-            __runner()
+            self.runner(
+                inv_always_validation_sequence_set,
+                inv_eventually_validation_sequence_set,
+                test_timeout_s[0] if type(test_timeout_s) == list else test_timeout_s,
+                tick_duration_s,
+            )
+
+
+class AggregateTestRunner(SimulatedTestRunner):
```

Comment:
put class docs for what an aggregatetestrunner is

Reply:
done